pkg/ui/src/redux/alerts.ts

```diff
@@ -50,6 +51,9 @@ export interface Alert extends AlertInfo {
   // ThunkAction which will result in this alert being dismissed. This
   // function will be dispatched to the redux store when the alert is dismissed.
   dismiss: ThunkAction<Promise<void>, AdminUIState, void>;
+  // Makes alert to be positioned in the top right corner of the screen instead of
+  // stretching to full width.
+  showAsAlert?: boolean;
```

Comment:
can this be a bit more specific? maybe say `showAsAlertMessage`. This is a bit vague and confusing since the word alert appears everywhere.